### ISSUE TITLE
Can't use collector.py to update data since 2020-09-25

### ISSUE THREAD

**Issue by sysy007uuu** [microsoft/qlib#1626]
2023-08-17 12:35:27.211 | WARNING  | data_collector.utils:wrapper:491 - _get_simple: 1 :get data error: 000001.sz--2020-01-01 00:00:00+08:00--2020-08-25 00:00:00+08:00The stock may be delisted, please check
2023-08-17 12:35:38.291 | WARNING  | collector:get_data_from_remote:147 - get data error: 000001.sz--2020-01-01 00:00:00+08:00--2020-08-25 00:00:00+08:00Your data request fails. This may be caused by your firewall (e.g. GFW). Please switch your network if you want to access Yahoo! data
2023-08-17 12:35:38.292 | WARNING  | data_collector.utils:wrapper:491 - _get_simple: 2 :get data error: 000001.sz--2020-01-01 00:00:00+08:00--2020-08-25 00:00:00+08:00The stock may be delisted, please check
2023-08-17 12:35:49.387 | WARNING  | collector:get_data_from_remote:147 - get data error: 000001.sz--2020-01-01 00:00:00+08:00--2020-08-25 00:00:00+08:00Your data request fails. This may be caused by your firewall (e.g. GFW). Please switch your network if you want to access Yahoo! data
2023-08-17 12:35:49.388 | WARNING  | data_collector.utils:wrapper:491 - _get_simple: 3 :get data error: 000001.sz--2020-01-01 00:00:00+08:00--2020-08-25 00:00:00+08:00The stock may be delisted, please check
2023-08-17 12:36:00.467 | WARNING  | collector:get_data_from_remote:147 - get data error: 000001.sz--2020-01-01 00:00:00+08:00--2020-08-25 00:00:00+08:00Your data request fails. This may be caused by your firewall (e.g. GFW). Please switch your network if you want to access Yahoo! data
**i have already close my firewall.**

**Comment by sysy007uuu** [comment]
![image](https://github.com/microsoft/qlib/assets/34132715/14d416f6-b6bc-44b3-8cce-1c819e4ef110)

**Comment by sysy007uuu** [comment]
I have set up a global proxy using Proxifier, but still can't access it.

**Comment by SunsetWolf** [comment]
I understand that this problem is due to network, even using global proxy doesn't work, I am using windows, setting the port of proxy server in cmd solves the problem, run the following command in cmd:
`set HTTP_PROXY=http://<your_proxy_server_address>:<your_proxy_port>`
`set HTTPS_PROXY=http://<your_proxy_server_address>:<your_proxy_port>`
With `yahooquery.Ticker(<symbol_name>, asynchronous=False).history(interval=<interval>, start=<start>, end=<end>)` you can fetch the data in cmd without enabling the global proxy.
Of course, using the collector script in qlib in cmd to fetch yahoo data is no problem.
![image](https://github.com/microsoft/qlib/assets/30293408/e0e2e175-bb99-4e35-8fe5-0f6fcc9bbc0c)

**Comment by danbo6** [comment]
> I have set up a global proxy using Proxifier, but still can't access it.

hi, do you have any updates about this issue? I'm in china mainland, and also met this issue.

**Comment by danbo6** [comment]
> I understand that this problem is due to network, even using global proxy doesn't work, I am using windows, setting the port of proxy server in cmd solves the problem, run the following command in cmd: `set HTTP_PROXY=http://<your_proxy_server_address>:<your_proxy_port>` `set HTTPS_PROXY=http://<your_proxy_server_address>:<your_proxy_port>` With `yahooquery.Ticker(<symbol_name>, asynchronous=False).history(interval=<interval>, start=<start>, end=<end>)` you can fetch the data in cmd without enabling the global proxy. Of course, using the collector script in qlib in cmd to fetch yahoo data is no problem. ![image](https://user-images.githubusercontent.com/30293408/267874918-6169abaa-e0c0-49ec-84ec-fed0b72eca0b.png)

Hi, I tried this method, but still does not work. I'm using WSL on windows and not sure if it is the reason.

**Comment by SunsetWolf** [comment]
This solution is for the windows command prompt (cmd) and after my testing it works and I'm still using it normally. If you are using WSL, I'm guessing it would help if you added the proxy to the environment variable? I don't use WSL, this is just a guess.

**Comment by danbo6** [comment]
> This solution is for the windows command prompt (cmd) and after my testing it works and I'm still using it normally. If you are using WSL, I'm guessing it would help if you added the proxy to the environment variable? I don't use WSL, this is just a guess.

set HTTP_PROXY=http://<your_proxy_server_address>:<your_proxy_port>
set HTTPS_PROXY=http://<your_proxy_server_address>:<your_proxy_port>

when you run these two commands, what is the  "your_proxy_server_address:your_proxy_port", does this mean your local ip, 127.0.0.1 or some other ip?

**Comment by SunsetWolf** [comment]
Yes, you are right, it depends on the configuration of the ip proxy.

**Comment by danbo6** [comment]
> set HTTP_PROXY=http://<your_proxy_server_address>:<your_proxy_port> set HTTPS_PROXY=http://<your_proxy_server_address>:<your_proxy_port>

i got no luck, still met this error: 

requests.exceptions.ProxyError: HTTPSConnectionPool(host='query2.finance.yahoo.com', port=443): Max retries exceeded with url: /v6/finance/quoteSummary/aapl?modules=summaryDetail&formatted=false&lang=en-US&region=US&corsDomain=finance.yahoo.com (Caused by ProxyError('Unable to connect to proxy', OSError(0, 'Error')))

**Comment by danbo6** [comment]
> I understand that this problem is due to network, even using global proxy doesn't work, I am using windows, setting the port of proxy server in cmd solves the problem, run the following command in cmd: `set HTTP_PROXY=http://<your_proxy_server_address>:<your_proxy_port>` `set HTTPS_PROXY=http://<your_proxy_server_address>:<your_proxy_port>` With `yahooquery.Ticker(<symbol_name>, asynchronous=False).history(interval=<interval>, start=<start>, end=<end>)` you can fetch the data in cmd without enabling the global proxy. Of course, using the collector script in qlib in cmd to fetch yahoo data is no problem. ![image](https://user-images.githubusercontent.com/30293408/267874918-6169abaa-e0c0-49ec-84ec-fed0b72eca0b.png)

it workes. Thanks!
